terraform/aws/security/aws-elasticsearch-nodetonode-encryption.tf

```diff
@@ -0,0 +1,73 @@
+# pass
+resource "aws_elasticsearch_domain" "without_cluster_config" {
+  domain_name           = "without_cluster_config"
+}
+
+resource "aws_elasticsearch_domain" "without_instance_count" {
+  domain_name           = "without_instance_count"
+
+  cluster_config {}
+}
+
+resource "aws_elasticsearch_domain" "instance_count_not_bigger_than_1" {
+  domain_name           = "instance_count_not_bigger_than_1"
+
+  cluster_config {
+    instance_count = 1 // a value <= 1
+  }
+}
+
+resource "aws_elasticsearch_domain" "node_to_node_encryption_enabled" {
```

Comment:
Should this one be flagged? It's currently highlighted: https://semgrep.dev/s/5yqX

Reply:
This shouldn't be flagged. Need to work on this to identify what is the issue